packages/amazonq/src/lsp/chat/activation.ts

```diff
@@ -12,24 +12,26 @@ import { Commands, getLogger, globals, undefinedIfEmpty } from 'aws-core-vscode/
 import { activate as registerLegacyChatListeners } from '../../app/chat/activation'
 import { DefaultAmazonQAppInitContext } from 'aws-core-vscode/amazonq'
 import { AuthUtil, getSelectedCustomization } from 'aws-core-vscode/codewhisperer'
-import {
-    DidChangeConfigurationNotification,
-    updateConfigurationRequestType,
-} from '@aws/language-server-runtimes/protocol'
+import { pushConfigUpdate } from '../config'
 
 export async function activate(languageClient: LanguageClient, encryptionKey: Buffer, mynahUIPath: string) {
     const disposables = globals.context.subscriptions
 
     // Make sure we've sent an auth profile to the language server before even initializing the UI
-    await pushConfigUpdate(languageClient, {
-        type: 'profile',
-        profileArn: AuthUtil.instance.regionProfileManager.activeRegionProfile?.arn,
-    })
-    // We need to push the cached customization on startup explicitly
-    await pushConfigUpdate(languageClient, {
-        type: 'customization',
-        customization: getSelectedCustomization(),
-    })
+    //
+    // Ideally the handler for onDidChangeRegionProfile would trigger this, but because the handler is set up too late (due to the current structure), the initial event is missed.
+    // So we have to explicitly do it here on startup.
+    if (AuthUtil.instance.isConnectionValid()) {
```

Comment:
should this be moved up to the main activation? I guess we don't really need to bury them in here anymore